src/main/kotlin/com/exactpro/th2/common/metrics/CommonMetrics.kt

```diff
@@ -21,6 +21,12 @@ import io.prometheus.client.Gauge
 import java.util.concurrent.CopyOnWriteArrayList
 import java.util.concurrent.atomic.AtomicBoolean
 
+/**
+ * This is a set of default baskets in millisecond for the Prometheus histogram metric.
```

Comment:
This is a set of default baskets in millisecond for **Prometheus** histogram metric.